nautobot/core/api/metadata.py

```diff
@@ -316,21 +339,20 @@ def restructure_view_config(self, view_config):
         """
 
         # TODO(timizuo): Add a standardized way of handling `tenant` and `tags` fields, Possible should be on last items on second col.
-        fields_to_remove = ["composite_key", "url", "display", "status", "id"]
-        fields_to_add = ["id", "composite_key", "url"]
+        fields_to_remove = ["composite_key", "url", "display", "status", "id", "created", "updated"]
+        advanced_fields = self.get_advanced_tab_fields(serializer)
         # Make a deepcopy to avoid altering view_config
         view_config_layout = deepcopy(view_config.get("layout"))
 
-        for section_idx, section in enumerate(view_config_layout):
-            for idx, value in enumerate(section.values()):
+        for _, section in enumerate(view_config_layout):
+            for _, value in enumerate(section.values()):
                 for field in fields_to_remove:
                     if field in value["fields"]:
                         value["fields"].remove(field)
-                if section_idx == 0 and idx == 0:
-                    value["fields"].extend(fields_to_add)
-
         if view_config.get("include_others", False):
-            view_config_layout = self.add_missing_field_to_view_config_layout(view_config_layout, fields_to_remove)
+            view_config_layout = self.add_missing_field_to_view_config_layout(
+                view_config_layout, fields_to_remove, advanced_fields
+            )
```

Comment:
```suggestion
            view_config_layout = self.add_missing_field_to_view_config_layout(
                view_config_layout, fields_to_remove + advanced_fields
            )
```

Since `advanced_fields` is part of the fields to be removed, this approach might be better than explicit adding a new attr to `self.add_missing_field_to_view_config_layout`